packages/tabs/src/Tabs.spec.tsx

```diff
@@ -60,6 +56,30 @@ describe('packages/tabs', () => {
   });
 
   describe('rendering', () => {
+    test('consoles errors if multiple tabs have the same name', () => {
```

Comment:
What if they have different HTML, but the same text content (i.e. `<span>Third</span>` vs `<div>Third</div>`? I think this should work, but can we add a test?